roundabout-core/src/test/java/at/fhv/itm3/s2/roundabout/entity/StreetSectionTest.java

```diff
@@ -71,15 +123,38 @@ public void isEnoughSpace() throws Exception {
         double streetSectionLengthBigger = 10.0;
         when(streetSectionMock.getLength()).thenReturn(streetSectionLengthBigger);
         assertTrue(streetSectionMock.isEnoughSpace(car.getLength()));
+    }
+
+    @Test
+    public void isEnoughSpace_equals() throws Exception {
+        IStreetSection streetSectionMock = mock(StreetSection.class);
+        ICar car = mock(Car.class);
+        when(car.getLength()).thenReturn(4.5);
+        HashMap<ICar, Double> carPositions = new HashMap<>();
+        carPositions.put(car, 3.5);
+        when(streetSectionMock.getCarPositions()).thenReturn(carPositions);
+        when(streetSectionMock.getLastCar()).thenReturn(car);
+        when(streetSectionMock.isEnoughSpace(car.getLength())).thenCallRealMethod();
 
         double streetSectionLengthEquals = 8.0;
         when(streetSectionMock.getLength()).thenReturn(streetSectionLengthEquals);
         assertFalse(streetSectionMock.isEnoughSpace(car.getLength()));
+    }
+
+    @Test
+    public void isEnoughSpace_smaller() throws Exception {
+        IStreetSection streetSectionMock = mock(StreetSection.class);
+        ICar car = mock(Car.class);
+        when(car.getLength()).thenReturn(4.5);
+        HashMap<ICar, Double> carPositions = new HashMap<>();
```

Comment:
please add a empty line before a new part, e.g. in this case before creating the car positions
do this the same for other parts in the tests